code/src/styledComponents/Input.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+
+
+export const Input = styled.input`
```

Comment:
To add placeholder styling you can do this:
```
&::placeholder {
    color: red;
  }
``` 
almost the same as hover, but with two ::

Reply:
Big thanks @JennieDalgren for a super useful code review, and that you took time to do it, even if the project was handed in late! 
And thanks for a fantastic spring, learning a  lot and enjoying it 95% of the time! :-D Not to mentio the remaining 5%, where you and your colleagues are there to help and support!
Have lovely maternity leave, and have a fantastic spring and summer with your family! :)

Reply:
Love it, big thanks! :)